Templates/ManualTrigger-CSharp/sample.dat

```diff
@@ -0,0 +1 @@
+Foobar
```

Comment:
Let's not use "Foobar". How about something like "Test Value".

Also, I don't see a sample.dat for the Node ManualTrigger template?